### ISSUE TITLE
🐛 [BUG] lammps/build/lmp: No such file (tutorial)

### ISSUE THREAD

**Issue by paulelenord** [mir-group/nequip#261]
**Describe the bug**
In colab tutorial example running LAMMPS fails at finding ''../lammps/build/lmp'':
```
!cd lammps_run/ && ../lammps/build/lmp -in toluene_minimize.in
>>>/bin/bash: ../lammps/build/lmp: No such file or directory
```

**To Reproduce**
Run the colab notebook with max_epochs 200

**Expected behavior**
LAMMPS runs

**Comment by Linux-cpp-lisp** [comment]
Hi @paulelenord ,

Thanks for your interest in our code!

Are you sure you successfully finished running the first two cells of the tutorial, which compile LAMMPS? You can remove the `%%capture` line at the top of the cells if you want to see the output and check that they are working. Note that compiling LAMMPS in Colab can take some time, usually around 8 minutes depending on the hardware they allocate.

**Comment by ZKC19940412** [comment]
Hi, I ran into the same issues. It looks like on LAMMPS end, the stable version of lammps points to the one at June 2022, and it is somewhat incompatible with the complication with nequip.  It changed line of lammps complie such that it fetches the lammps at Sep 2021 and it seems to work for me consistently afterwards.

I also attached a png on how I modify the complie. Hope this helps.

Best, 
Zekun
![lammps_nequip_complation_modifies](https://user-images.githubusercontent.com/27995442/202832579-f2630ab7-74a3-461d-bd5e-0605a4cf47a0.PNG)
